src/Parser/Core/Modules/Features/SpellInfo.js

```diff
@@ -72,6 +72,27 @@ export default {
     mastery: false,
     vers: true,
   },
+  [SPELLS.XAVARICS_MAGNUM_OPUS.id]: { // Prydaz
+    int: false,
+    crit: false,
+    hasteHpct: false,
+    mastery: false,
+    vers: true,
+  },
+  [SPELLS.HEALTHSTONE.id]: {
+    int: false,
+    crit: false,
+    hasteHpct: false,
+    mastery: false,
+    vers: false, // not 100% sure
+  },
+  [SPELLS.MARK_OF_THE_ANCIENT_PRIESTESS.id]: {
+    int: false,
+    crit: true,
+    hasteHpct: false,
```

Comment:
Does Mark have any haste PPM scaling? Because if so it should probably have `hasteHpct: true`

Reply:
No

Reply:
Unless both my source and Wowhead lied